docs/user_api/index.rst

```diff
@@ -3,19 +3,33 @@
 User API
 ========
 
-This page will be automatically populated once one or more
-Uxarray user API functions are implemented.
-
-For now, please check the
-tentative `Uxarray API
+This page shows already-implemented Uxarray user API functions. You can also
+check the draft `Uxarray API
 <https://github.com/UXARRAY/uxarray/blob/main/docs/user_api/uxarray_api.md>`_
-documentation and let us know if you have any feedback!
+documentation to see the tentative whole API and let us know if you have any feedback!
+
+Grid Class
+----------
+.. autosummary::
+   :toctree: _autosummary
 
-Routines
---------
+   grid.Grid
 
-Uxarray native routines
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Grid functions
+--------------
+.. autosummary::
+   :toctree: _autosummary
+
+   grid.Grid.saveas_file
+   grid.Grid.write
+   _exodus.read_exodus
+   _exodus.write_exodus
+   _ugrid.write_ugrid
+   _ugrid.read_ugrid
 
+Helper functions
+----------------
 .. autosummary::
-   :toctree: ./generated/
+   :toctree: _autosummary
+
+   helpers.determine_file_type
```

Comment:
I think we intended to provide this helper function to be callable through `uxarray.grid.determine_file_type`, right? If so, I believe the documentation should show it through the same path to instruct the user correct. Can we please check if that's possible?

Reply:
Grid doesn't have to be loaded to `determine_file_type`, so I made it accessible directly via uxarray as `uxarray.determine_file_type`

Reply:
Sounds good!